src/statistics.cc

```diff
@@ -0,0 +1,177 @@
+// Copyright 2016 Ismael Jimenez Martinez. All rights reserved.
```

Comment:
I'm not sure about this copyright header. It was copied from `src/complexity.cc`
But i'll just add myself here...